### DISCUSSION TITLE
PDF.js Issue 2 - Feature in "View PDF" for drawing tool

### DISCUSSION THREAD

**Discussion by hotcoding85** [hotcoding85/pdf.js#3] [Q&A] [answered]
Hi, can you add a rubber for delete the writings?
But instead of deleting all, just a part (similar to Samsung notes)

Thanks

**Comment by po1206** [comment, marked as answer]
For future reference: Please remember to provide a more descriptive title, since it's not really possible to understand what the issue is about when reading the title.
Furthermore, please actually fill out the ISSUE_TEMPLATE properly (rather than ignoring/deleting it) such that all relevant information is available.
